src/predict_request_validation_utils.cpp

```diff
@@ -367,9 +368,15 @@ Status RequestValidator::validate() {
     return finalStatus;
 }
 
+template <>
 Status validate(const tensorflow::serving::PredictRequest& request, const tensor_map_t& inputsInfo, const std::string& servableName, const model_version_t servableVersion, const std::set<const char*>& optionalAllowedInputNames, const Mode batchingMode, const shapes_info_map_t& shapeInfo) {
     return RequestValidator(request, inputsInfo, servableName, servableVersion, optionalAllowedInputNames, batchingMode, shapeInfo).validate();
 }
 
+template <>
+Status validate(const ::inference::ModelInferRequest& request, const tensor_map_t& inputsInfo, const std::string& servableName, const model_version_t servableVersion, const std::set<const char*>& optionalAllowedInputNames, const Mode batchingMode, const shapes_info_map_t& shapeInfo) {
```

Comment:
If KFS has no namespace. maybe we could add it? kfs::inference::ModelInferRequest

Reply:
kfs has ::inference namespace